readthedocs-theme/static/img/sponsors/sponsor-logo-light.svg

```diff
@@ -0,0 +1,67 @@
+	c0.3,0.4,0.6,0.6,1,0.8c0.4,0.2,0.9,0.3,1.5,0.3s1.1-0.1,1.5-0.3c0.4-0.2,0.7-0.5,1-0.8c0.3-0.3,0.4-0.8,0.6-1.3
+	c0.1-0.5,0.2-1,0.2-1.6V390c0-0.6,0-1.2-0.2-1.6c-0.1-0.5-0.3-0.9-0.6-1.3c-0.3-0.4-0.6-0.6-1-0.8c-0.4-0.2-0.9-0.3-1.5-0.3
+	s-1.1,0.1-1.5,0.3c-0.4,0.2-0.7,0.5-1,0.8c-0.3,0.3-0.4,0.8-0.6,1.3c-0.1,0.5-0.2,1-0.2,1.6V399.7z M308.8,406.1l-5.8-15v15h-3.2
+	v-22.6h2.4l5.6,14v-14h3.2v22.6H308.8z M320.4,406.3c-0.8,0-1.5-0.1-2.1-0.3c-0.6-0.2-1.2-0.4-1.6-0.8s-0.8-0.7-1.2-1.1
+	c-0.3-0.4-0.6-0.8-0.8-1.2c-0.2-0.5-0.4-1-0.6-1.5c-0.1-0.6-0.2-1.2-0.3-1.8l3.2-0.8c0,0.5,0.1,1,0.2,1.6c0.1,0.6,0.3,1.1,0.5,1.5
+	c0.3,0.5,0.6,0.9,1,1.2c0.4,0.3,0.9,0.5,1.5,0.5c1.6,0,2.4-0.8,2.4-2.4c0-0.4,0-0.8-0.2-1.1c-0.1-0.3-0.3-0.7-0.5-1
+	s-0.5-0.6-0.8-0.9c-0.3-0.3-0.6-0.6-1.1-0.9l-4-3.2c-0.4-0.3-0.8-0.7-1.1-1.1c-0.3-0.4-0.6-0.9-0.8-1.3c-0.2-0.5-0.3-0.9-0.4-1.4
+	c-0.1-0.5-0.1-0.9-0.1-1.4c0-0.7,0.1-1.4,0.3-2.1c0.2-0.6,0.6-1.2,1.1-1.7c0.5-0.5,1.1-0.9,1.8-1.2c0.7-0.3,1.5-0.4,2.5-0.4
+	c0.8,0,1.5,0.1,2.1,0.3c0.6,0.2,1.2,0.4,1.6,0.7c0.5,0.3,0.9,0.6,1.2,0.9c0.3,0.3,0.6,0.7,0.8,1.1c0.2,0.4,0.4,0.9,0.6,1.4
+	c0.1,0.5,0.2,1,0.3,1.5l-3.2,0.8c0-0.4-0.1-0.9-0.2-1.3c-0.1-0.5-0.3-0.9-0.5-1.3c-0.3-0.4-0.6-0.7-1-1c-0.4-0.3-0.9-0.4-1.5-0.4
+	c-0.4,0-0.8,0.1-1.1,0.2c-0.3,0.1-0.6,0.3-0.7,0.5c-0.2,0.2-0.3,0.5-0.4,0.8c-0.1,0.3-0.1,0.6-0.1,0.9c0,0.6,0.1,1.1,0.4,1.7
+	c0.3,0.6,0.7,1.1,1.2,1.5l4,3.2c0.5,0.4,1,0.9,1.4,1.3c0.4,0.5,0.7,1,1,1.5c0.3,0.6,0.5,1.1,0.6,1.7c0.2,0.6,0.2,1.1,0.2,1.7
+	c0,0.7-0.1,1.4-0.3,2c-0.2,0.6-0.6,1.2-1,1.7c-0.5,0.5-1,0.9-1.8,1.2C322.1,406.2,321.3,406.3,320.4,406.3z M341.1,398.8
+	c0,1-0.1,2-0.3,2.9s-0.6,1.7-1.1,2.4c-0.5,0.6-1.2,1.2-2,1.6c-0.8,0.4-1.8,0.6-3.1,0.6c-0.9,0-1.6-0.1-2.2-0.3
+	c-0.6-0.2-1.2-0.4-1.7-0.8c-0.5-0.3-0.9-0.7-1.2-1.2s-0.6-1-0.8-1.5c-0.2-0.6-0.3-1.2-0.4-1.8c-0.1-0.6-0.1-1.3-0.1-1.9v-8.1
+	c0-0.7,0-1.3,0.1-1.9c0.1-0.6,0.2-1.2,0.4-1.8c0.2-0.6,0.5-1.1,0.8-1.5c0.3-0.5,0.7-0.9,1.2-1.2s1-0.6,1.7-0.8
+	c0.6-0.2,1.4-0.3,2.2-0.3c1.2,0,2.3,0.2,3.1,0.6c0.8,0.4,1.5,0.9,2,1.6s0.9,1.5,1.1,2.4c0.2,0.9,0.3,1.8,0.3,2.9V398.8z
+	 M331.4,399.7c0,0.6,0,1.2,0.2,1.6c0.1,0.5,0.3,0.9,0.6,1.3c0.3,0.4,0.6,0.6,1,0.8s0.9,0.3,1.5,0.3s1.1-0.1,1.5-0.3
+	c0.4-0.2,0.7-0.5,1-0.8c0.3-0.3,0.4-0.8,0.6-1.3c0.1-0.5,0.2-1,0.2-1.6V390c0-0.6,0-1.2-0.2-1.6s-0.3-0.9-0.6-1.3
+	c-0.3-0.4-0.6-0.6-1-0.8s-0.9-0.3-1.5-0.3s-1.1,0.1-1.5,0.3c-0.4,0.2-0.7,0.5-1,0.8c-0.3,0.3-0.4,0.8-0.6,1.3
+	c-0.1,0.5-0.2,1-0.2,1.6V399.7z M352.8,406.1l-3.3-10.5h-2.4v10.5h-3.2v-22.6h4c1.4,0,2.6,0.1,3.5,0.4c0.9,0.3,1.7,0.7,2.2,1.2
+	c0.6,0.6,0.9,1.2,1.2,1.9c0.2,0.7,0.3,1.5,0.3,2.5c0,0.9-0.1,1.7-0.3,2.3c-0.2,0.6-0.4,1.1-0.7,1.5c-0.3,0.4-0.6,0.7-0.9,0.9
+	c-0.3,0.2-0.6,0.4-0.9,0.5l3.6,11.3L352.8,406.1L352.8,406.1z M347.1,393.2h0.8c0.6,0,1.2,0,1.6-0.2s0.9-0.3,1.3-0.6
+	c0.4-0.3,0.6-0.6,0.8-1.1c0.2-0.5,0.3-1,0.3-1.8c0-0.7-0.1-1.3-0.3-1.8s-0.5-0.8-0.8-1.1c-0.3-0.3-0.8-0.5-1.3-0.6
+	c-0.5-0.1-1-0.2-1.6-0.2h-0.8V393.2z"/>
+</svg>

```

Comment:
So, we do have actual logos available, we should start thinking about bringing those logos in here. Do you want to do that now or in a follow up PR?

I suppose if we want to come back to this, it might make sense to make individual graphic files on this PR for each sponsor. That is, copy this placeholder file to separate files so then we just need to come back and update the svg files later.

Reply:
I think it would be easier to iterate on this later. I understand that means merging code that is not ready for production and that might be a problem.

The changes made in this PR, however, are just about translating HTML mockups into the Pelican templates, so it might make sense to review just the pelican syntax and/or file structure and open separate PR's for revisiting contents, copywriting, links and UX?

Reply:
Sure, though we do already have all of these graphics available already. Was there a reason we couldn't use them?

Reply:
There's no other reason. I'll add them now. The footer contents were imported here from the app when we decided to revert back to what was already done there and we just hadn't iterate on them again since.

Reply:
It would also be great if we could get our hands in the `.svg` version of these logos! That would allow better quality on scaling but also simplify any color adjustments we might need to do.

Reply:
Also, looks like we can remove this file (`sponsor-logo-light.svg`) as it won't be used anywhere, correct?

Reply:
Noted below, but David is the one that originally added the image files for sponsors. You could ask if he had source SVGs here, or take a look through the sponsor media kits eventually as well.